src/main/webapp/js/user-page-loader.js

```diff
@@ -173,6 +180,10 @@ function buildMessageDiv(message) {
       ' [' + message.sentimentScore + ']'));
 
   const bodyDiv = document.createElement('div');
+  if(message.imageURL){
+    bodyDiv.innerHTML += '<br/>';
+    bodyDiv.innerHTML += '<img src="' + message.imageURL + '" />';
+  }
   bodyDiv.classList.add('message-body');
   bodyDiv.innerHTML = message.text;
```

Comment:
This line resets the `innerHTML` of the `bodyDiv`, which basically undoes the `<img>` element you built on line 185. This is why you aren't seeing images. I think you probably want to move this line so it's above the `if` statement.